libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-row-delete.directive.ts

```diff
@@ -110,20 +119,22 @@ export class SkyAgGridRowDeleteDirective
             overlay.componentRef.instance.updateClipPath(clipPath);
           });
 
-        setTimeout(() => {
-          const inlineDeleteRef = this.#rowDeleteComponent?.inlineDeleteRefs
-            ?.toArray()
-            .find(
-              (elRef) => elRef.nativeElement.id === 'row-delete-ref-' + id,
-            ) as ElementRef;
-          const affixer = this.#affixService.createAffixer(inlineDeleteRef);
-
-          this.#affixToRow(affixer, id);
-
-          this.#rowDeleteContents[id] = {
-            affixer: affixer,
-            overlay: overlay,
-          };
+        runInInjectionContext(this.#environmentInjector, () => {
+          afterNextRender(() => {
```

Comment:
`afterNextRender` is still in developer preview for 19, are we ok with using it?